src/RapidezStatamicServiceProvider.php

```diff
@@ -4,27 +4,55 @@
 
 use Illuminate\Support\ServiceProvider;
 use TorMorten\Eventy\Facades\Eventy;
-use Statamic\Facades\GlobalSet;
+
+use Rapidez\Statamic\Commands\SyncProductsCommand;
+use Rapidez\Statamic\Commands\SyncStoresCommand;
+use Statamic\Facades\Entry;
+use Statamic\Stache\Repositories\EntryRepository as StatamicEntryRepository;
+use Rapidez\Statamic\Repositories\EntryRepository;
+use Statamic\Statamic;
 use Illuminate\Support\Facades\View;
+use Illuminate\View\View as RenderedView;
+use Statamic\Facades\GlobalSet;
 use Statamic\Facades\Site;
+use Validator;
 
 class RapidezStatamicServiceProvider extends ServiceProvider
 {
     public function boot()
     {
-        $this->bootRoutes()
-        ->bootGlobals();
+        $this->bootCommands()
+            ->bootRepositories()
+            ->bootPublishables()
+            ->bootFilters()
+            ->bootComposers();
     }
 
-    public function bootRoutes() : self
+    public function bootFilters() : self
     {
         Eventy::addFilter('routes', fn ($routes) => array_merge($routes ?: [], [__DIR__.'/../routes/fallback.php']));
 
         return $this;
     }
 
-    public function bootGlobals() : self
+
+    public function bootCommands() : self
     {
+        $this->commands([
+            SyncProductsCommand::class,
+            SyncStoresCommand::class
+        ]);
+
+        return $this;
+    }
+
+    public function bootComposers() : self
+    {
+        View::composer('rapidez::product.overview', function (RenderedView $view) {
+            $entry = Entry::whereCollection('products')->where('sku', config('frontend.product')['sku'])->first();
+            $view->with('content', $entry);
```

Comment:
Is this always wanted/needed? Can imagine someone only wants the products in Statemic so it can be related to other collections instead of replenishing the product with fields from Statemic.